src/gui/org/deidentifier/arx/gui/resources/Resources.java

```
@@ -286,6 +287,20 @@ public InputStream getStream(final String name) {
     
     /**
      * Returns an image.
+     * 
+     * @param type
+     * @return
+     */
+    public Image getImage(AttributeType type) {
+        if (type == AttributeType.IDENTIFYING_ATTRIBUTE) { return getManagedImage("bullet_red.png"); }
+        if (type == AttributeType.INSENSITIVE_ATTRIBUTE) { return getManagedImage("bullet_green.png"); }
+        if (type == AttributeType.SENSITIVE_ATTRIBUTE) { return getManagedImage("bullet_purple.png"); }
+        if (type == AttributeType.QUASI_IDENTIFYING_ATTRIBUTE) { return getManagedImage("bullet_yellow.png"); }
+        return null;

```

Please throw an IllegalArgumentException("Unknown attribute type"); here.